src/main/java/stepsdesign/PipelineSteps.java

```diff
@@ -165,13 +165,13 @@ public void previewAndRunThePipeline() {
     CdfStudioLocators.runButton.click();
   }
 
-  @Then("Preview and run the pipeline with runtime arguments")
+  @Then("Enter runtime argument value {string} for key {string}")
+  public void enterRuntimeArgumentValueForKey(String value, String runtimeArgumentKey) {
+    CdfStudioLocators.runtimeArgsValue(runtimeArgumentKey).sendKeys(PluginPropertyUtils.pluginProp(value));
+  }
+
+  @Then("Run the preview of pipeline with runtime arguments")
   public void previewAndRunThePipelineWithRuntimeArguments() {
-    CdfStudioActions.previewSelect();
-    CdfStudioActions.clickPreviewRunButton();
-    for (Map.Entry<String, String> entry : runtimeArguments.entrySet()) {
-      CdfStudioLocators.runtimeArgsValue(entry.getKey()).sendKeys(entry.getValue());
-    }
     CdfStudioActions.clickConfigRunButton();
```

Comment:
This locator and action need to be revisited. Action name should be `clickPreviewConfigRunBtn()` and similarly locator name should be `previewConfigRunBtn`.